reader/pom.xml

```diff
@@ -10,8 +15,15 @@
     <name>Kaazing Metrics Reader</name>
     <url>https://github.com/kaazing/metrics</url>
 
+    <scm>
+        <connection>scm:git:${project.scm.url}</connection>
+        <developerConnection>scm:git:${project.scm.url}</developerConnection>
+        <url>git@github.com:kaazing/metrics.git</url>
+    </scm>
+
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jmock.version>2.6.0</jmock.version>
```

Comment:
This should be defined in the parent pom.xml instead of duplicating it each submodule pom.